### ISSUE TITLE
IllegalArgumentException when maxBackoff = firstBackoff

### ISSUE THREAD

**Issue by b-dzoba** [reactor/reactor-addons#176]
https://github.com/reactor/reactor-addons/blame/master/reactor-extra/src/main/java/reactor/retry/Backoff.java#L90

**Comment by simonbasle** [comment]
Indeed. I think it makes better sense to let `maxBackoff` be `==` to `firstBackoff` (as the message points to).
